crates/opencascade/examples/chamfer.rs

```diff
@@ -0,0 +1,39 @@
+use glam::dvec3;
+use opencascade::{
+    primitives::{Direction, Face, Solid},
+    workplane::Workplane,
+};
+
+pub fn main() {
+    // A tapering chamfer from bottom to top 2->1
+    let mut base = Workplane::xy().rect(10.0, 10.0);
+    base.chamfer(2.0);
+
+    let mut top = Workplane::xy().rect(10.0, 10.0);
+    top.translate(dvec3(0.0, 0.0, 10.0));
+    top.chamfer(1.0);
+
+    let chamfered_box = Solid::loft([&base, &top].into_iter()).to_shape();
+
+    // insert the workplane into the chamfered box area so union returns edges
+    let handle = Workplane::xy().translated(dvec3(0.0, 0.0, 0.1)).rect(5.0, 5.0);
```

Comment:
In the end I guess we're still not able to get unioned edges properly when the faces are in the same plane?